boards/nano33ble_rev2/README.md

```diff
@@ -0,0 +1,282 @@
+Arduino Nano 33 BLE Sense Rev2
+===================
+
+<img src="https://media.elektor.com/media/catalog/product/cache/9cc822bfc6a57f9729d464b8b5e0e0df/a/r/arduino-nano-33-ble-sense-rev2-with-headers_front.jpg" width="35%">
+
+The [Arduino Nano 33 BLE Sense Rev2](https://store.arduino.cc/products/nano-33-ble-sense-rev2) is a 
+compact board based on the Nordic nRF52840 SoC. It includes the following sensors:
+
+- 3 axis geomagnetic sensor
+- 6 axis inertial sensor
+- humidity and temperature sensor
+- barometric sensor
+- microphone
+- gesture, proximity, light color and light intensity sensor
+
+
+## Getting Started
+
+First, follow the [Tock Getting Started guide](../../doc/Getting_Started.md).
+
+The Nano 33 comes pre-installed with a
+[version](https://github.com/arduino/ArduinoCore-nRF528x-mbedos/tree/master/bootloaders/nano33ble)
+of the BOSSA bootloader that Arduino uses on various boards. Unfortunately this
+bootloader is not well suited for Tock development. Specifically, it doesn't
+support reading from the board, so there is no way to automatically determine
+what type of board it is or what is already installed. It's also [not open
+source](https://github.com/arduino/ArduinoCore-nRF528x-mbedos/issues/23) (at
+least as of December 2020).
```

Comment:
Can we update this at all? Is this still true 3 years later?

Reply:
I think that its still the same, correct me if I'm wrong.